backend/tests/apps/slack/models/google_auth_test.py

```
@@ -0,0 +1,311 @@
+from apps.slack.models.google_auth import GoogleAuth
+from apps.slack.models.member import Member
+
+
+class TestGoogleAuthModel:
+    """Test cases for GoogleAuth model."""
+
+    @pytest.fixture(autouse=True)
+    def setUp(self):
+        """Set up test data."""
+        self.member = Member(slack_user_id="U123456789", username="testuser")
+        self.valid_token = "valid_access_token"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "valid_token"

(S105)